tests/publiccloud/transfer_repos.pm

```diff
@@ -35,6 +35,8 @@ sub run {
         $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec sed -i 's,http://,/tmp/repos/repos/,g' '{}' \\;");
         $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec zypper ar '{}' \\;");
         $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec echo '{}' \\;");
+
+        $args->{my_instance}->run_ssh_command(cmd => "sudo zypper lr");
```

Comment:
this command can be done by regular user you don't need sudo

Reply:
Will this do the trick?
```perl
$args->{my_instance}->run_ssh_command(cmd => "sudo zypper lr", username => $username);
```

Reply:
I think this was ment
```suggestion
        $args->{my_instance}->run_ssh_command(cmd => "zypper lr");
```

Reply:
> I think this was ment

exactly

Reply:
Sorry, didn't see that `sudo` 😄